babel.config.js

```diff
@@ -1,3 +1,12 @@
 module.exports = {
-  presets: ["@vue/app"]
+  presets: [
```

Comment:
How will this affect the Babel compilation of the web application in production mode? It might be better to have two different babel configs for unit testing and for production/development work. What is your opinion on this?

Reply:
I've updated the babel.config.js with two environments so the main build will use the "@vue/app" preset and only the test command will use the new one